components/pedigree/resources/src/main/resources/pedigree/model/export.js

```diff
@@ -347,11 +348,14 @@ define([
                  var cancerData = pedigree.GG.properties[i].cancers[cancer];
                  if (!cancerData.affected) {
                      output += "0\t";
+                     if (yob == "0") {
+                        warnMissingDOBUnaff = true;
+                     }
                  } else {
                      var ageAtDetection = cancerData.hasOwnProperty("numericAgeAtDiagnosis") && (cancerData.numericAgeAtDiagnosis > 0)
                                           ? cancerData.numericAgeAtDiagnosis : "AU";
                      output += ageAtDetection.toString() + "\t";
-                     if (yob == "0") {
+                     if (yob == "0" || ageAtDetection == "AU") {
```

Comment:
Ah, right you are. After some testing, BOADICEA will accept "AU" (even if a yob is given) and reject whenever a yob isn't given for affected individuals. Will also warn if yob is not given for unaffected individuals. Will change the warnings to mirror that.